system/Router/RouteCollection.php

```diff
@@ -625,7 +625,7 @@ public function group(string $name, ...$params)
         // To register a route, we'll set a flag so that our router
         // so it will see the group name.
```

Comment:
I think the "so it"  can be removed the way the previous line is phrased.

Reply:
@sfadschm Oh, thanks! I didn't understand the sentence.